go/tools/runquiet/runquiet.go

```diff
@@ -50,9 +195,26 @@ func main() {
 		},
 	}
 	fmt.Printf("Launching %s with args %v\n", os.Args[argsIndex], os.Args[argsIndex:])
-	pid, _, err := syscall.StartProcess(os.Args[argsIndex], os.Args[argsIndex:], attr)
+
+	var err error
+	var pid int
+
+	if elevated {
+		token, _ := getUserToken()
+
+		pid, _, err = CreateProcessWithTokenW(token, os.Args[argsIndex:], attr)
+		defer syscall.CloseHandle(syscall.Handle(token))
+		if err != nil {
+			fmt.Printf("CreateProcessWithTokenW error: %s\n", err.Error())
+			return err
+		}
+	} else {
+		pid, _, err = syscall.StartProcess(os.Args[argsIndex], os.Args[argsIndex:], attr)
+	}
+
 	if err != nil {
 		fmt.Printf("StartProcess error: %s\n", err.Error())
+		return err
```

Comment:
early exit here intended? old code would still test `doWait`

Reply:
Also, probably want to return the error in the `err != nil` check below too, right?

Reply:
returning the error seems ok, but the old code would skip the doWait in the error case, wouldn't it? Return codes were less important here when this wasn't invoked from RunWithPrivilege